### DISCUSSION TITLE
app is lagging

### DISCUSSION THREAD

**Discussion by ducthanh200398** [wailsapp/wails#2350] [Q&A]
im running wails.exe with reacjs table, at 5000 line of data it start lagging, and the frontend disappear at 10k line

**Comment by leaanthony** [comment]
For that level of frontend usage, I'd suggest using something that doesn't use a virtual dom, like svelte. From what I've seen, react doesn't scale.

**Reply by ducthanh200398** [reply]
ty, it stoped lagging.

**Comment by ZeronoFreya** [comment]
You show too many nodes, you can take a look at this https://github.com/bvaughn/react-virtualized.
